bundles/org.openhab.binding.meater/src/main/resources/OH-INF/i18n/meater.properties

```diff
@@ -0,0 +1,63 @@
+# binding
+
+binding.meater.name = MEATER Binding
+binding.meater.description = This is the binding for MEATER probe.
+
+# thing types
+
+thing-type.meater.meaterapi.label = MEATER Cloud REST API
+thing-type.meater.meaterapi.description = This bridge represents the MEATER Cloud REST API connector
+thing-type.meater.meaterprobe.label = MEATER Probe
+thing-type.meater.meaterprobe.description = This thing represents a MEATER Probe
+
+# thing types config
+
+thing-type.config.meater.meaterapi.email.label = Email
+thing-type.config.meater.meaterapi.email.description = The email used to login to MEATER Cloud account
+thing-type.config.meater.meaterapi.password.label = Password
+thing-type.config.meater.meaterapi.password.description = The password used to login to MEATER Cloud account
+thing-type.config.meater.meaterapi.refresh.label = Refresh Interval
+thing-type.config.meater.meaterapi.refresh.description = Specifies the refresh interval in seconds
+thing-type.config.meater.meaterprobe.deviceId.label = Device Id
+thing-type.config.meater.meaterprobe.deviceId.description = Unique id for your MEATER Probe
+
+config.missing-username-password.description = Configuration of username and password is mandatory
+config.negative-refresh-interval.description = Refresh time cannot be negative!
+
+# channel types
+
+channel-type.meater.ambientTemperature.label = Probe Ambient Temperature
+channel-type.meater.ambientTemperature.description = Ambient temperature reading of MEATER Probe. If ambient is less than internal, ambient will equal internal
+channel-type.meater.cookElapsedTime.label = Cook Elapsed Time
+channel-type.meater.cookElapsedTime.description = Time since the start of cook in seconds. Default: 0
+channel-type.meater.cookEstimatedEndTime.label = Cook Estimated End Time
+channel-type.meater.cookEstimatedEndTime.description = Date and time of estimated end time for current cook
+channel-type.meater.cookEstimatedEndTime.state.pattern = %1$tY-%1$tm-%1$td %1$tH:%1$tM
+channel-type.meater.cookId.label = Current Cook ID
+channel-type.meater.cookId.description = Unique cook ID of current cook
+channel-type.meater.cookName.label = Current Cook Name
+channel-type.meater.cookName.description = Name of selected meat in your language or user given custom name
+channel-type.meater.cookPeakTemperature.label = Current Cook Peak Temperature
+channel-type.meater.cookPeakTemperature.description = Peak temperature of current cook
+channel-type.meater.cookRemainingTime.label = Cook Remaining Time
+channel-type.meater.cookRemainingTime.description = Remaining time in seconds or UNDEF when unknown
```

Comment:
In the XML this description ends with punctuation, so the file should be regenerated by running command `mvn i18n:generate-default-translations`.

Reply:
Regenerated!